client/lib/application/view_application/view_application_bloc.dart

```diff
@@ -0,0 +1,70 @@
+import 'dart:async';
+import 'package:bloc/bloc.dart';
+import 'package:client/domain/application/application.dart';
+import 'package:client/domain/application/application_failure.dart';
+import 'package:client/domain/application/i_application_repository.dart';
+import 'package:client/domain/application/value_objects.dart';
+import 'package:client/infrastructure/application/application_highlight_dto.dart';
+import 'package:client/injectable.dart';
+import 'package:dartz/dartz.dart';
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:injectable/injectable.dart';
+import 'package:meta/meta.dart';
+part 'view_application_event.dart';
+part 'view_application_state.dart';
+
+part 'view_application_bloc.freezed.dart';
+
+@lazySingleton
+class ViewApplicationBloc
+    extends Bloc<ViewApplicationEvent, ViewApplicationState> {
+  ViewApplicationBloc() : super(ViewApplicationState.initial());
+
+  @override
+  Stream<ViewApplicationState> mapEventToState(
+    ViewApplicationEvent event,
+  ) async* {
+    final IApplicationRepository _iApplicationRepository =
+        getIt<IApplicationRepository>();
+    yield* event.map(
+      started: (e) async* {
+        // Start Loader animation
+        yield state.copyWith(isFetching: true);
+
+        // Mock Api call
+        await Future.delayed(const Duration(seconds: 2));
```

Comment:
what is its purpose?

Reply:
Umm Will remove it.. was just to check the loaders functionality